build-conf/README.md

```diff
@@ -41,7 +41,7 @@ applicationDefaultPropFiles | Comma separated list of default application confi
 buildListFileExt | File extension that indicates the build file is really a build list.
 applicationConfRootDir | Alternate root directory for application-conf location.  Allows for the deployment of the application-conf directories to a static location.  Defaults to ${workspace}/${application}
 createBuildOutputSubfolder | Option to create a subfolder with the build label within the build output dir (outDir). Default: true.
-buildOutputTSformat | Defines the build timestamp format for build output subfolder and build label.
+(Deprecated) buildOutputTSformat | Defines the build timestamp format for build output subfolder and build label.
```

Comment:
@M-DLB Let's completely drop the `buildOutputTSformat` property. It is no longer used in your implementation anyway.

Reply:
Alright, but it should be part of the changelog that this property no longer exists.